sdk/attestation/azure-security-attestation/inc/azure/attestation/attestation_client.hpp

```diff
@@ -0,0 +1,410 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// SPDX-License-Identifier: MIT
+
+#pragma once
+
+#include <shared_mutex>
+#include <string>
+
+#include "attestation_client.hpp"
+#include "attestation_client_models.hpp"
+#include "attestation_client_options.hpp"
+
+namespace Azure { namespace Core { namespace Http { namespace _internal {
+  class HttpPipeline;
+}}}} // namespace Azure::Core::Http::_internal
+
+namespace Azure { namespace Security { namespace Attestation {
+  /** @brief An AttestationTokenHeader represents common properties in an the RFC 7515 JSON Web
+   * Token.
+   */
+  struct AttestationTokenHeader
```

Comment:
move to models?

Reply:
It's not strictly a model type, should it still go in models?